packages/augur-tools/src/templates-template.ts

```diff
@@ -390,6 +408,31 @@ function dateNoWeekend(
     ) {
       return false;
     }
+
+    const closing = closingDateDependencies[0];
```

Comment:
redundant code in templates-template and in UI validations. pull out the redundant code into a method that both can call.